finetune/base.py

```diff
@@ -177,6 +177,24 @@ def finetune(self, Xs, Y=None, batch_size=None):
 
         with warnings.catch_warnings():
             warnings.simplefilter("ignore")
+            if self.config.prefit_init:
+                tf.logging.info("Starting pre-fit initialisation...")
+                num_layers_trained = self.config.num_layers_trained
+                self.config.num_layers_trained = 0
+                estimator.train(train_input_fn, hooks=train_hooks, steps=num_steps)
```

Comment:
There's some room here for debating whether or not train_hooks / num_steps should be shared or whether this portion should always work the same way.

Reply:
For now we're going to leave this as is.